### PR TITLE
Develop feature/apply requests

### PR DESCRIPTION
**Pull request by sence1roandayne**
- Survey
     - [ ] Place "I want to subscribe" checkbox before the Submit button
     - [ ] Add classification, e.g. parent, student
     - [ ] Remove the "You are subscribed to our newsletter as student" since there is a validation in place
- Navigation
     - [ ] Change "Search" to "Search Scholarship"
- Add Scholarship
     - [ ] Remove "+" for benefits, requirements, and eligibilities
     - [ ] Add sample answers
     - [ ] Add cancel to delete prompt
- Signin/Signup
     - [ ] Remove "OR"

### CODE REVIEW

File: frontend/src/components/DashboardTable/DashboardTable.tsx [hunk 1]
```diff
@@ -148,21 +148,37 @@ export default function DataTable() {
...
+                <Button
+                  color="primary"
+                  onClick={() => setIsSnackbarOpen(false)}
+                  sx={{ alignSelf: 'center' }}
+                >
+                  Cancel
+                </Button>
+                <Button
+                  color="inherit"
+                  onClick={handleDelete}
```
Comment (christian-barrios995) [inline frontend/src/components/DashboardTable/DashboardTable.tsx:176]: delete button does not work. It shows the error: DELETE http://localhost:5001/api/v1/scholarships/6 422 (Unprocessable Content)

File: frontend/src/containers/SurveyPage/SurveyPage.tsx [hunk 2]
```diff
@@ -213,81 +233,95 @@ const SurveyPage: React.FC<SurveyPageProps> = ({ user_type }) => {
...
+              </Link>{' '}
+              document
+            </Typography>
+          )}
+        </Box>
+      )}
+      <Button
+        variant="contained"
+        color="secondary"
+        onClick={handleSubmit}
```
Comment (christian-barrios995) [inline frontend/src/containers/SurveyPage/SurveyPage.tsx:320]: image
I get the same error message as dashboard deleting

**Review by christian-barrios995 — APPROVED** [review]
the UI LGTM! 
I have few errors on my end, might be only on my end.

**Comment by sence1roandayne** [comment]
> the UI LGTM! I have few errors on my end, might be only on my end.

@christian-barrios995 Thanks! Pushed the fixes

**Pull request merged**